src/lib/picker.ts

```diff
@@ -705,16 +705,25 @@ class PickerLoader {
 
   private _initialized: Promise<PickerInstance>;
 
+  private client: Client;
+
+  private pickerOptions: PickerOptions;
+
   constructor(client: Client, options?: PickerOptions) {
     const validateRes = getValidator(PickerParamsSchema)(options);
     if (validateRes.errors.length) {
       throw new FilestackError(`Invalid picker params`, validateRes.errors, FilestackErrorType.VALIDATION);
     }
 
     this._initialized = this.loadModule(client, options);
+
+    this.client = client;
+    this.pickerOptions = options;
   }
 
   async open(): Promise<void> {
+    await this.client.prefetch(this.pickerOptions);
```

Comment:
we have to check if we need to do it in this pleace, maybe move it to init or call it manualy from picker.